kafkautil/formatters_test.go

```diff
@@ -0,0 +1,67 @@
+package kafkautil
+
+import (
+	"testing"
+
+	kafka "github.com/confluentinc/confluent-kafka-go/v2/kafka"
+	"github.com/spf13/viper"
+)
+
+func TestFmtMessageId(t *testing.T) {
+	type args struct {
+		offset kafka.Offset
+		delay  int64
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
```

Comment:
this is exactly how I structure my test, nice!

Reply:
VScode test generate function was a great find. I wish I could take credit though.